src/settings/BatchGroupConfigurationSettings/BatchGroupConfigurationSettings.js

```diff
@@ -45,6 +49,24 @@ const BatchGroupConfigurationSettings = ({ mutator }) => {
   // eslint-disable-next-line react-hooks/exhaustive-deps
   []);
 
+  const fetchBatchVoucherExports = useCallback(
+    (id) => {
+      if (id) {
+        setBatchVoucherExports();
+
+        mutator.batchVoucherExports.GET({
+          params: {
+            query: `batchGroupId==${id} sortby end/sort.descending start/sort.descending`,
```

Comment:
please add limit param
```
limit: `${LIMIT_MAX}`,
```

Reply:
mb batch load is better here? - I don't know how many exports can be for batch group

Reply:
hmm, good point. it might be even a better idea to use pagination and Load more button, since API should support it

Reply:
do you mean `onNeedMoreData` for MCL?

Reply:
I can add this to your PR, so you don't have to spend time on it

Reply:
I would be appreciated. Thanks